src/beautifiers/index.coffee

```diff
@@ -70,6 +70,7 @@ module.exports = class Beautifiers extends EventEmitter
     'yapf'
     'erl_tidy'
     'marko-beautifier'
+    'r-beautifier'
```

Comment:
Ok, I just checked and with **formatR** in src/beautifiers/index.html and src/beautifiers/r-beautifier/index.html the `npm run docs` fails with `Error: Cannot find module './formatR'`